app/Policies/AlbumQueryPolicy.php

```diff
@@ -71,7 +71,7 @@ public function applyVisibilityFilter(AlbumBuilder|FixedQueryBuilder $query): Al
 			$query2
 				->where(
 					fn (AlbumBuilder|TagAlbumBuilder $q) => $q
-						->where('base_albums.requires_link', '=', false)
+						->where('base_albums.is_link_required', '=', false)
```

Comment:
I am not going to check every occurrence of `requires_link` myself. I assume that you did a search+replace and spotted all occurrences.